src/Components/FormCreation/CreateEditForm.js

```diff
@@ -93,14 +93,14 @@ function CreateEditForm() {
   async function deleteSection() {
     // call API to delete
     const response = FORM.deleteSection(
-      loadForm.value.formId,
+      loadForm.value._id,
       loadForm.value.sections[activeSection]._id
     )
       .then(() => {
         //setDeletedSection({...loadForm.value.sections[activeSection]});
 
         dispatchSectionsUpdate({
-          type: "DELETE_SECTION",
+          type: "LOAD",
```

Comment:
Does "LOAD" make sense here? Why was delete section action not working?

Reply:
This was done in way of moving from permanently deleting a section to just toggling a field for each section called "deleted". I will rework "DELETE_SECTION" when I finish with undoing deleting a section, but for now I made this PR so that Greg can get the changes we made while working on this.

Reply:
Sounds good, thanks for clarifying!